code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -266,14 +266,14 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	ignitermes = "<span class='notice'>USER fiddles with FLAME, and manages to light their NAME with the power of science.</span>"
 
 /obj/item/clothing/mask/smokable/cigarette/cigar/cohiba
-	name = "\improper Cohiba Robusto cigar"
+	name = "cohiba robusto cigar"

```

Comment:
These two words, plus Havanian, should have capitalized letters. They're names, brand names.